docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+# specify keyboard / keymap / subproject (if req) and run `docker compose up`
+# will install all deps into a docker container and build firmware, cross platform
+# Only prereq is docker
+qmk:
+  build: .
+  environment:
+    keyboard: lets_split
+    keymap: adam
+    subproject: rev2
+  volumes:
+    - .:/qmk
```

Comment:
fair enough, I like it as I don't have to remember all the params for docker, but that just might be me.  Removed.